modules/@apostrophecms/schema/ui/apos/scss/AposInputArray.scss

```diff
@@ -265,6 +263,7 @@
 
   &.collapse-enter-active,
   &.collapse-leave-active {
+    overflow: hidden;
```

Comment:
We only need the overflow hidden during the animation. This way it's not the case when opened and the color picker can overflow.